packages/runtime/src/index.ts

```diff
@@ -10,7 +10,13 @@ export interface RuntimeOptions {
 declare global {
   interface Window {
     __unocss?: UserConfig & { runtime?: RuntimeOptions }
-    __unocss_runtime?: { uno: UnoGenerator; extractAll: () => void; version: string }
+    __unocss_runtime?: {
+      uno: UnoGenerator
+      extractAll: () => void
+      inspect: (callback?: (element: Element) => boolean) => void
+      toggleObserver: (state?: boolean) => void
```

Comment:
Can you add some jsdocs for those APIs? Thanks

Reply:
PR #247